softwerkskammer/lib/commons/fieldHelpers.js

```diff
@@ -1,13 +1,19 @@
 /* eslint no-underscore-dangle: 0 */
 
 const moment = require('moment-timezone');
-const numeral = require('numeral');
-numeral.language('de', require('numeral/languages/de'));
-numeral.language('en-gb', require('numeral/languages/en-gb'));
 
 const conf = require('simple-configure');
 const misc = conf.get('beans').get('misc');
 
+// adding additional languages to builtin Intl
+if (!require('intl-locales-supported')(['de', 'en-gb'])) {
+  // `Intl` exists, but it doesn't have the data we need, so load the
+  // polyfill and replace the constructors with need with the polyfill's.
```

Comment:
Den darfst Du trotzdem rechtschreibkorrigieren ;-)

Ich bezog mich auf den zweiten Satz: "and replace the constructors with need" -> sollte wohl heißen "we need" ?!?